docker-compose.yml

```diff
@@ -31,6 +31,8 @@ services:
       - "workspace"
       - "@zooniverse/fe-content-pages"
     command: ["start"]
+    environment:
+      - ASSET_PREFIX=http://localhost:3000
```

Comment:
I'm testing out #1563 at the moment and I'm testing on `http://localhost.zooniverse.org:3000` and `http://localhost.zooniverse.org:3001` so that the Panoptes API works. Does the hostname make a difference here?

Reply:
It probably should match to avoid cross origin issues and best practice would be to use `local.zooniverse.org:3001`. In theory this would work as the this prefix is valid as the content pages app is running on `http://localhost:3001` on the host machine.

I'm probably going to close this PR as i don't have time to work on it, it does however solve the issue of why the content pages app doesn't work from root docker-compose setup.

Reply:
Yeah, I added this line to my set-up for testing #1563 and it worked just fine. I don't see any project icons on the publications page but I think that's a separate issue.